core/src/scss/utilities/variables/core/z-index.scss

```diff
@@ -0,0 +1,34 @@
+@charset "UTF-8";
+
+//
+// Z-index
+//
+// Style guide: Variables.Core.Z-index
+//
+
+//
+// $level
+//
+// A sass map of z-indices
+//
+//```
+// $z-levels: (
+//   "navigation":  100,
+//   "button":      100,
+//   "masthead":     20,
+//   "overlay":      10,
+//   "default":       1,
+//   "background":  -100
+// );
+// ```
+//
+// Style guide: Variables.Core.Z-index.level
+//
+$z-levels: (
+  "navigation":  100,
+  "button":      100,
+  "masthead":     20,
+  "overlay":      10,
+  "default":       1,
+  "background":  -100
+);
```

Comment:
Are we adding the main nav z-indexes in here? Another good one to add might be the skiplinks (focus state) since that needs to be above everything in the masthead when it comes into focus. Looks like a great start 😄 👍 - thanks for doing this much needed work!

Reply:
@yvonnetangsu we could! I was wondering how many we wanted to call out explicitly, and how many we might want to make relative (so for example z-index: z(navigation) - 1). I can see benefits to either approach.

Reply:
> @yvonnetangsu we could! I was wondering how many we wanted to call out explicitly, and how many we might want to make relative (so for example z-index: z(navigation) - 1). I can see benefits to either approach.

Yeah, that's a good question. It might be nice to have one on the map that's really high up - like main nav uses 9999 as default I believe, since we want it above most everything else. The drop down menus on the main nav has higher z-index, something like 11111 because in case someone put another main nav right under, we want the expanded drop down for the top main nav to cover up the bottom main nav, and then the lockup within the masthead has some value in between the two. So maybe main nav 9999 is another good one to be on the map - we can do relative values for the drop down and the lockup then.

Reply:
And most of the time when I try to assign z-index values to two items, I set the numbers at least 10 apart in case I decide to add something else later that's in between.

Reply:
So in thinking about this some more, it feels like the landmarks are the stuff we know we have (main nav, drop-downs, masthead, etc) and that if we define those relative to each other, and far enough apart to allow for multiple relative values to be set, people will have the ability to easily set values for things they add in (like a second main nav, etc.) The changes in my next push will reflect that approach.